sql/moz-fx-data-shared-prod/telemetry/mobile_usage_2021/view.sql

```diff
@@ -0,0 +1,33 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.mobile_usage_2021`
+AS
+WITH base AS (
+  SELECT
+    *,
+    IF(product LIKE '% iOS', 'iOS', 'Android') AS os,
```

Comment:
Do you have an example of a product name? Do they always have a space in between?

Reply:
> The query looks fine to me, but I don't have much context on what problem this is solving. Do you have a related bug, issue, or ticket?

Jeff walked me through this the other day, its helping us build the official corporate KPI dashboards for this year in looker.

Reply:
The values that are in the table now always have a space and iOS apps always end in iOS so this should work.

Reply:
See https://mozilla.github.io/bigquery-etl/mozfun/norm/#product_info-udf for the table of product names. This is definitely a bit hacky, but is a stopgap until we get more fully-formed baseline ETL with historical first_seen_date, etc. I will add more commentary to the query.